resources/css/style.css

```diff
@@ -442,10 +476,10 @@ input:not([type]), input[type=text], input[type=password], input[type=email], in
   opacity: 0.7;
 }
 ul:not(.browser-default) {
```

Comment:
This will affect all the uls, so we can do this only with the ul with class page-footer